test/jesse_tests_draft3_SUITE.erl

```diff
@@ -145,3 +146,13 @@ remoteRefExtra(Config) ->
 
 unicodePatternProperties(Config) ->
   do_test("unicodePatternProperties", Config).
+
+uniqueItemsExtra(Config) ->
+  do_test("uniqueItemsExtra", Config).
+
+uniqueItemsPerformance(Config) ->
+  Start = erlang:system_time(millisecond),
+  do_test("uniqueItemsPerformance", Config),
+  End = erlang:system_time(millisecond),
+  ElapsedMilliseconds = End - Start,
+  ?assert(ElapsedMilliseconds < 100).
```

Comment:
oh, this has a risk of being unstable if CI agent is not fast enough... Do you think there is enough margin for it to not be flaky?

Reply:
I see it failed in CI several times because it took more than 100ms. Also, would be nice to log the `ElapsedMilliseconds` to see how much it have been exceeded

```suggestion
  ?assert(ElapsedMilliseconds < 100, [{elapsed, ElapsedMilliseconds}]).
```
but I don't remember if `?assert/2` is supported on older OTP versions

Reply:
I think it is a bit ugly to have this performance tests within a CT suite.
I thought it would be safe to put it with 100 ms. since in my laptop it takes below 20 ms to run the tests with the improvement and around 800 ms without the improvement. But I see now that it doesn't pass for the CI workers.
If it is ok with you, I will remove it.